docs/getting-started/zowe-secure-setup.md

```diff
@@ -1,59 +1,158 @@
+PEM and PKCS#7 formats use Base64 ASCII encoding while DER and PKCS#12 use binary encoding.
+
+In general, the choice of certificates format depends on the technologies used for the implementation of the server components and on the certificate storage type. 
+
+
+Zowe supports:
+* **file-based PKCS12**  
+  PKCS12 certificates are the most general and widely deployed certificate format.
+* **z/OS keyring-based keystore (JKS/JCEKS)**  
+  JKS/JCEKS certificates are specific types of certificates that depend on the Java environment.
+
+**Note:** Java 9 and higher can also work with PKCS12 certificates.
+
+### Certificates storage
+
+There are two options for the storage of certificates:
+
+* Keystore and Truststore Combination
+* SAF Keyrings
+
+#### Keystore and Truststore Combination
 
-You can choose to use a SAF keyring instead of keystore and truststore for storing certificates. It is more secure than PKCS12 files. And SAF keyring allows you to import existing or generate new certificates with Top Secret, ACF2, and RACF.
+Two key concepts to understand storage and verification of certificates are keystores and truststores.
 
-For details about SAF Keyring, see the documentation [here](../extend/extend-apiml/certificate-management-in-zowe-apiml.md).
+* **Keystores** are used to store certificates and the verification of these certificates.
+* **Truststores** are used for the storage of verification.
 
+Zowe supports keystores and truststores that are either z/OS keyrings (when on z/OS) or PKCS12 files. By default, Zowe reads a PKCS12 keystore from `keystore` directory in zowe.yaml. This directory contains a server certificate, the Zowe generated certificate authority, and a `truststore` which holds intermediate certificates of servers that Zowe communicates with (for example z/OSMF).
+
+* **Keystores**  
+  Zowe can use PKCS12 certificates stored in USS to encrypt TLS communication between Zowe clients and Zowe z/OS servers, as well as intra z/OS Zowe server to Zowe server communication. Zowe uses a `keystore` directory to contain its external certificate, and a `truststore` directory to hold the public keys of servers which Zowe communicates with (for example z/OSMF).
+
+* **Truststores**  
+  Truststore are essential to provide secure communication with external services. The truststore serves as a secure repository for storing certificates and trust anchors. In the context of Zowe, the truststore establishes the trust relationships with external services as well as manages the relationship between Zowe's components and the certificates presented by the external services.
+
+  In addition to utilizing the intra-address space of certificates, Zowe incorporates external services on z/OS to enhance the encryption of messages transmitted between its servers. These external services, such as z/OSMF or Zowe conformant extensions, have registered themselves with the API Mediation Layer.
```

Comment:
Maybe a link to z/osmf  doc would be helpful here

Reply:
Same here. Will add links. Thanks!